src/stdlib/camelcase.rs

```diff
@@ -0,0 +1,134 @@
+use crate::compiler::prelude::*;
+
+use convert_case::{Case, Casing};
+
+fn camelcase(value: Value, orig_case: Option<Case>) -> Resolved {
+    match orig_case {
+        Some(case) => Ok(value
+            .try_bytes_utf8_lossy()?
+            .from_case(case)
+            .to_case(Case::Camel)
+            .into()),
+        None => Ok(value.try_bytes_utf8_lossy()?.to_case(Case::Camel).into()),
+    }
+}
+
+pub(crate) fn variants() -> Vec<Value> {
+    vec![
+        crate::value!("camelCase"),
+        crate::value!("PascalCase"),
+        crate::value!("SCREAMING_SNAKE"),
+        crate::value!("snake_case"),
+        crate::value!("kebab-case"),
+    ]
+}
+
+pub(crate) fn into_case(s: String) -> Result<Case, Box<dyn DiagnosticMessage>> {
+    match s.as_ref() {
+        "camelCase" => Ok(Case::Camel),
+        "PascalCase" => Ok(Case::Pascal),
+        "SREAMING_SNAKE" => Ok(Case::ScreamingSnake),
+        "snake_case" => Ok(Case::Snake),
+        "kebab-case" => Ok(Case::Kebab),
+        _ => Err(Box::new(ExpressionError::from(
+            "must match either camelCase PascalCase, SCREAMING_SNAKE snake_case or kebab-case"
```

Comment:
Nit you could use `variants()` here like `fmt!("must match one of {}", variants)`

Reply:
done